plugins/module_utils/kibana.py

```diff
@@ -854,16 +854,16 @@ def get_fleet_server_hosts(self):
     result = self.send_api_request(endpoint, 'GET')
     return result['item']['fleet_server_hosts']
 
-  def set_fleet_server_hosts(self, hosts: list):
-    endpoint = 'fleet/settings'
+  def set_fleet_server_hosts(self, hosts: list, name='Default', default=True):
+    endpoint = 'fleet/fleet_server_hosts/fleet-default-fleet-server-host' 
     headers = {'kbn-xsrf': True}
     body = {
-        'fleet_server_hosts': hosts
+        'is_default': default,
+        'name': name,
+        'host_urls': hosts
```

Comment:
does this only work past a certain version of fleet? it's fine if that's the case, we should just call it out in the release notes/readmes